segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/SegmentStoreIntegrationTest.java

```diff
@@ -45,50 +39,15 @@
     private static final int BOOKIE_COUNT = 3;
     private File baseDir = null;
     private MiniDFSCluster hdfsCluster = null;
-    private BookKeeperServiceRunner bkRunner;
-    private CuratorFramework zkClient;
+    private BookKeeperRunner helper = null;
 
     /**
      * Starts BookKeeper and HDFS MiniCluster.
```

Comment:
We have `SegmentStoreFileSystemIntegrationTest` and this one I suppose we should have called `SegmentStoreHDFSIntegrationTest` but there was no need at the time. What about renaming the classes to:

```
FileSystemIntegrationtest
HDFSIntergationTest
```

The `SegmentStore` part of the name is unnecessary as this is under `segmentstore/`.